src/idl_parser.cpp

```diff
@@ -1314,6 +1350,38 @@ CheckedError Parser::ParseSingleValue(Value &e) {
   return NoError();
 }
 
+void Parser::InitializeNestedFlatbufferParser(Parser& nestedParser, Value *const nested) {
+  assert(nested);
```

Comment:
What do further layers of nested_flatbuffers require to be copied i.o. to be parseable?